usaspending_api/etl/management/sql/swap_in_new_table/update_live_tables.py

```diff
@@ -0,0 +1,6 @@
+update_live_tables_sql = """
+UPDATE {dest_table} AS d
```

Comment:
Missing the `SET` phrase for an `UPDATE` query

Reply:
addressed